e2e2/test/cases/nvidia/manifests/mpi-job-nccl-test-multi-node.yaml

```diff
@@ -29,7 +29,7 @@ spec:
             - --allow-run-as-root
             - --tag-output
             - -np
-            - "16"
+            - "{{.GpuCount}}"
```

Comment:
shouldn't this be `replicas * requests` or `WorkerNodeCount * GpuCount`?

Reply:
You are right, will correct it.